WinUIGallery/Controls/ControlExample.xaml

```diff
@@ -80,7 +80,7 @@
             IsTextSelectionEnabled="True"
             Visibility="Collapsed">
             <Paragraph>
-                This sample requires a later version of Windows to be fully functional. Learn more about version adaptive apps:<Hyperlink NavigateUri="https://learn.microsoft.com/windows/uwp/debug-test-perf/version-adaptive-apps">https://learn.microsoft.com/windows/uwp/debug-test-perf/version-adaptive-apps</Hyperlink>
+                This sample requires a later version of Windows to be fully functional.
```

Comment:
Do we still need this or can we remove this feature? WinUI 3 isn't really platform agnostic anymore and I don't think we actually need to show a warning for samples that only work on specific OS versions.

Reply:
Your change looks good.  We should keep this check, which is based on the ApiContract versions.

Reply:
Sure let's keep it though I don't think we actually still use this. Do you happen to know what the replacement link would be or do we keep the changes as is and remove the info on how to do such a check?